nexus/types/src/deployment/planning_report.rs

```diff
@@ -502,16 +504,16 @@ impl PlanningMupdateOverrideStepReport {
 #[cfg_attr(test, derive(test_strategy::Arbitrary))]
 pub enum FailedMgsUpdateReason {
     /// There was a failed attempt to plan a Host OS update
-    #[error("failed to plan a Host OS update")]
+    #[error("failed to plan a Host OS update: {0}")]
```

Comment:
I had to put the "{0}" back as the full error wasn't being displayed :(

Reply:
This isn't the right change; this means we're hitting [Pitfall 1: Not Logging the Full Error Chain](https://github.com/oxidecomputer/omicron/blob/main/docs/error-types-and-logging.adoc#21-pitfall-1-not-logging-the-full-error-chain) which notes

> The easy but incorrect way to fix this is to change the outer error to include the source error.

Instead, we need to fix the place that's displaying the error and have it walk the full chain.

Reply:
I read the doc! Thanks a bunch for taking the time to write it :)